package.json

```diff
@@ -16,5 +16,11 @@
   "bugs": {
     "url": "https://github.com/gregtuc/Soen-341/issues"
   },
-  "homepage": "https://github.com/gregtuc/Soen-341#readme"
+  "homepage": "https://github.com/gregtuc/Soen-341#readme",
```

Comment:
Could you make this a generic, and not sue Greg's account?

Reply:
yes i'll change that